sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py

```diff
@@ -93,6 +93,7 @@ def test_torch_run_inference_imagenet_mobilenetv2(self):
 
   @pytest.mark.uses_pytorch
   @pytest.mark.it_postcommit
+  @pytest.mark.skip
```

Comment:
```suggestion
  @pytest.mark.skip # TODO(https://github.com/apache/beam/issues/22413): Debug input data issue and unskip
```

This should link to the relevant issue.